{
    "topic": "Error Causes",
    "pageNumber": "9",
    "type": "Unexpected Error",
    "definition": {
        "text": "An Unexpected Error is an error that happened at runtime that does not have a programmed way to handle it, and it usually produces the procedure where it happens to be halted. For those errors we preserve all the information available and display it at the Docs for users to analyze them.",
        "updated": 1611733132608,
        "translations": [
            {
                "language": "RU",
                "text": "Неожиданная ошибка - это ошибка, произошедшая во время выполнения, которая не имеет запрограммированного способа ее обработки, и обычно вызывает процедуру, в которой она останавливается. Для этих ошибок мы сохраняем всю доступную информацию и отображаем ее в документах Docs, чтобы пользователи могли их анализировать.",
                "updated": 1638553423837
            }
        ]
    },
    "paragraphs": [
        {
            "style": "Text",
            "text": "Right click and Edit to enter edit mode and change this text. ENTER to write new paragraphs. ESC to exit edit mode. Then you can use docs.save to save your changes and app.contribute to send it to the main repository.",
            "translations": [
                {
                    "language": "RU",
                    "text": "Щелкните правой кнопкой мыши и выберите Edit, чтобы войти в режим редактирования и изменить этот текст. ENTER, чтобы написать новые абзацы. ESC для выхода из режима редактирования. Затем вы можете использовать docs.save для сохранения изменений и app.contribute для отправки в основной репозиторий.",
                    "updated": 1638553439759
                }
            ]
        }
    ]
}